docs/README.md

```diff
@@ -1,14 +1,184 @@
-4. [doSub()](doSub)
-5. [doPositiveSub()](doPositiveSub)
-6. [doMul()](doMul)
-7. [doMulN()](doMulN)
-8. [doMod()](doMod)
-9. [calcAge()](calcAge)
-10. [calcLoan()](calcLoan)
-11. [timeCalc()](timeCalc)
-12. [getCurrency()](getCurrency)
+- [Introduction](#intro)
+- [doAdd()](#doAdd)
+- [doAddN()](#doAddN)
+- [doSub()](#doSub)
+- [doPositiveSub()](#doPositiveSub)
+- [doMul()](#doMul)
+- [doMulN()](#doMulN)
+- [doDiv()](#doDiv)
+- [doMod()](#doMod)
+- [doDecr()](#doDecr)
+- [doIncr()](#doIncr)
+- [doReverse()](#doReverse)
+- [toStr()](#toStr)
+- [toHex()](#toHex)
+- [toDec()](#toDec)
+- [toOct()](#toOct)
+- [toTri()](#toTri)
+- [toBin()](#toBin)
+- [rgbToHex()](#rgbToHex)
+- [hexToRgb()](#hexToRgb)
+- [calcAge()](#calcAge)
+- [calcLoan()](#calcLoan)
+- [timeCalc()](#timeCalc)
+- [getCurrency()](#getCurrency)
+
+
+#intro
+
+#doAdd
+
+function doAdd(num1, num2)
+
+Returns the sum of two numbers
+
+#doAddN
+
+function doAddN(nums)
+
+Returns the sum of the numbers in an array
+
+#doSub
+
+function doSub(num1, num2)
+
+Returns the subtraction of two numbers
+
+#doPositiveSub
+
+function doPositiveSub(num1, num2)
+
+Returns the positive subtraction of two numbers by checking if the first number is bigger or not
+
+#doMul
+
+function doMul(num1, num2)
+
+Multiplication of two numbers
+
+#doMulN
+
+function doMulN(nums)
+
+Multiplication of all numbers in the array
+
+#doDiv
+
+function doDiv(num1, num2)
+
+Division of two numbers
+
+#doMod
+
+function doMod(num1, num2)
+
+Modulus of two numbers
+
+#doDecr
+
+function doDecr(num)
+
+Returns number smaller by one from input
+
+#doIncr
+
+function doIncr(num)
+
+Returns number bigger by one from input
+
+#doReverse
+
+function doReverse(num)
+
+Inverts number (from positive to negative OR from negative to positive)
+
+#toStr
+
+function toStr(num)
+
+Converts number to string
+
+#toHex
+
+function toHex(num)
+
+Converts number to hexadecimal
+
+#toDec
+
+function toDec(num)
+
+Converts number (back) to decimal
+
+#toOct
+
+function toOct(num)
+
+Converts number to octal
+
+#toTri
+
+function toTri(num)
+
+Converts number to trinary (0, 1, 2)
+
+#toBin
+
+function toBin(num)
+
+Converts number to binary
+
+#rgbToHex
+
+function rgbToHex(r, g, b)
+
+Converts rgb color to hex
+
+#hexToRgb
+
+function hexToRgb(color)
+
+Converts hex color to rgb
+
+#calcAge
+
+function calcAge(date)
+
+Returns time pased between a date and current time
+
+#calcLoan
+
+function calcLoan(amount, rate, month)
+
+Calculatess total amount from month one to pay for a loan 
+
+<!-- whoopsie, this dev never did anything money related... this section to update l8er... -->
+
+#timeCalc
+
+function timeCalc(seconds)
+
+Converts number of seconds to more readable format (year, month, day, hour, minute, second)
+
+#getCurrency
+
+function getCurrency(currencyFrom, currencyTo, value)
+
+Converts currencies.
+
+List of supported ones:
+
+- USD
+- EUR
+- GBP
+- CNY
+- JPY
+- CAD
+- AUD
+- AED
+
+<!-- same as  calcLoan. send help !!! -->
+
+
```

Comment:
LGTM.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Reply:
=]